docs/other/guides/how-to-create-scaffolding-templates.md

```diff
@@ -0,0 +1,45 @@
+---
+id: creatingScaffoldingTemplates
+title: How to create Scaffolding Templates
+sidebar_label: Creating Scaffolding Templates
+---
+
+### Introduction
+As discussed in [Scaffolding](/docs/get-started/scaffolding), it is possible to save yourself a lot of time by using one of the default Scaffolding Templates.
+
+The great news are that you may also write your own templates! In this article we will go in depth about it.
```

Comment:
`The great thing is you can create your own templates`

Reply:
done